GDAXSharp/Shared/Utilities/AbstractJson.cs

```diff
@@ -0,0 +1,28 @@
+﻿using Newtonsoft.Json;
```

Comment:
@chrisw000  @dougdellolio Not really satisfied with this file. For the `Services` and `WebSocket` we need to do some de- and serialization. So I thought, make it abstract and inherit, so it's easy to use without duplicate code. Perhaps a `Shared` class would be better?

Reply:
internal static class, with internal static methods to serialize / deserialize?